x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/hooks/use_hosts_table.tsx

```diff
@@ -64,10 +65,31 @@ export type HostNodeRow = HostMetadata &
  * Helper functions
  */
 const formatMetric = (type: InfraAssetMetricType, value: number | undefined | null) => {
-  return value || value === 0 ? createInventoryMetricFormatter({ type })(value) : 'N/A';
+  const defaultValue = value ?? 0;
+  return createInventoryMetricFormatter({ type })(defaultValue);
+};
+
+const buildMetricCell = (
+  value: number | null,
+  formatType: InfraAssetMetricType,
+  hasSystemMetrics?: boolean
+) => {
+  if (!hasSystemMetrics && value === null) {
+    return <AddDataTroubleshootingPopover />;
+  }
+
+  return formatMetric(formatType, value);
 };
 
 const buildItemsList = (nodes: InfraAssetMetricsItem[]): HostNodeRow[] => {
+  nodes.map((node) => {
+    if (node.name === 'instance') {
```

Comment:
Q: Why do we need this extra mapping? Do we need that because of the synthtrace hosts?

Reply:
Omg, I left this as part of some testing in local that I was doing, going to open a small PR to fix this.

Reply:
Here's the PR https://github.com/elastic/kibana/pull/191572!